Lesson Plans/Flight Instructor/CFI/CFI1.1 Human Behavior.md

```diff
@@ -0,0 +1,126 @@
+			3. It is often helpful to remind them *their own stated goals* for seeking aviation training.
+			4. Learning plateaus are a common source of frustration, discouragement — They should be encouraged to continue to work hard and be reassured that *it will pass and that the results will follow.*
+4. Human Needs
+	1. ![[Hierarchy of Needs.png]]*Maslow's Hierarchy of Needs* until the needs are satisfied, one cannot focus on learning.
+	2. **Physiological**: Instructors should monitor the learners to make sure their basic physical needs are met (not hungry, dehydrated, tired, etc.)
+	3. **Security**: Instructor should be aware of the learner's fear of flying and ease them into it carefully.
+	4. **Belonging**: Instructor should welcome the learners into the field of aviation, be it a career or hobby for learners.
+	5. **Esteem**: Self-esteem can come from internal judgement but most people seek external esteem through social approval (status, recognition, respect, etc.) It might not only be a strong influence on the instructor-learner relationship, it may be the main reason for a learner's interest in aviation training.
+	6. **Cognitive and Aesthetic**: *This layer is added in the later years and thus not in the picture shown above.* Cognitive is the need to know and understand, to control and make informed decisions. Aesthetic are emotional needs, if an instructor does not like a learner, that subtle feeling may leak into their teaching.
+	7. **Self-Actualization**: Realizing one's own potential for continued development / personal goals.
+	8. *Help learners satisfy their needs in a manner that creates a healthy learning environment.*
+5. Defense Mechanisms
+	1. **Definition**: Subconscious mental process to protect oneself from anxiety, unpleasant emotions, or to provide a refuge from a situation with which one cannot currently cope.
+		1. They often appear unconsciously.
+		2. They tend to distory, transform, or otherwise falsify reality.
+	2. **Displacement**: Unconscious shift of emotion, affect, or desire from the original object to a more acceptable, less threatening substitute.
+	3. **Denial**: Refusal to accept external reality because it is too threatening.
+		1. **Minimization**: Sometimes one will appear to have accepted it but in a diluted form.
+	4. **Rationalization**: Subconsciously justifying actions that otherwise would be unacceptable.
+	5. **Compensation**: Psychologically counterbalancing perceived weakness by emphasizing strength in other areas.
+	6. **Projection**: Places their own unacceptable impulses onto someone else, relegates the blame to others.
+	7. **Reaction Formation**: One fakes a belief opposite to the true belief (fact / what has happened).
+	8. **Fantasy**: Engages in daydreams about how things should be rather than doing anything about how things are.
+	9. **Repression**: One places uncomfortable thoughts into inaccessible ares of the unconscious mind.
+6. Learner Emotional Reactions
```

Comment:
The long lists of definitions starts to be a bit mind-numbing. Maybe break it into sections (H4 heading I think) and have examples for some of them?